assets/airlines/aircraft.json

```diff
@@ -0,0 +1 @@
:[["A306",3],["A319",4],["A320",17],["B735",2],["B736",7]]}},{"icao":"tay","name":"TNT Airways","callsign":{"name":"Quality","length":3},"fleets":{"default":[["B733",2],["B734",12],["B752",2],["B77L",3],["B744",4]]}},{"icao":"tcf","name":"Shuttle America","callsign":{"name":"Mercury","length":4},"fleets":{"default":[["E145",42],["E170",60]]}},{"icao":"tcx","name":"Thomas Cook Airlines","callsign":{"name":"Kestrel","length":3},"fleets":{"long":[["B752",1],["B753",2],["B763",3],["A332",6]],"short":[["A321",21]],"default":[["B752",1],["B753",2],["B763",3],["A332",6],["A321",21]]}},{"icao":"tfl","name":"TUI Airlines Netherlands","callsign":{"name":"Orange","length":3},"fleets":{"default":[["B738",8],["B763",1],["B788",3]]}},{"icao":"tgw","name":"Tigerair","callsign":{"name":"Go Cat","length":3},"fleets":{"default":[["A319",2],["A320",36],["B738",3]]}},{"icao":"thy","name":"Turkish Airlines","callsign":{"name":"Turkish","length":4},"fleets":{"default":[["A319",14],["A320",29],["A321",62],["B738",90],["B739",15],["E190",3]],"long":[["A332",20],["A333",30],["A343",4],["B77W",29]]}},{"icao":"tom","name":"Thomson Airways","callsign":{"name":"Tomson","length":3,"alpha":true},"fleets":{"long":[["B752",14],["B788",9],["B763",3]],"short":[["B738",31]],"default":[["B738",31],["B752",14],["B788",9],["B763",3]]}},{"icao":"tra","name":"Transavia Holland","callsign":{"name":"Transavia","length":4},"fleets":{"default":[["B737",8],["B738",30]]}},{"icao":"tso","name":"Transoviet Airlines","callsign":{"name":"Transoviet","length":3},"fleets":{"default":[["B733",2],["B734",4],["B735",10],["B738",12],["B744",9],["B762",2],["B763",8],["B772",2],["b77l",6],["B773",4],["T204",3]]}},{"icao":"tui","name":"TUIfly","callsign":{"name":"Tui Jet","length":4,"alpha":true},"fleets":{"default":[["B737",4],["B738",27]]}},{"icao":"tvf","name":"Transavia France","callsign":{"name":"France Soleil","length":4},"fleets":{"default":[["B738",27]]}},{"icao":"tvs","name":"Travel Service","callsign":{"name":"SKYTRAVEL","length":3},"fleets":{"default":[["A320",3],["B738",17],["B739",2]]}},{"icao":"twa","name":"Trans World Airlines - TWA","callsign":{"name":"TWA","length":3},"fleets":{"90long":[["L101",30],["B762",20],["B752",30],["B762",2]],"90default":[["MD81",150],["B722",100]]}},{"icao":"tya","name":"NordStar","callsign":{"name":"Taimyr","length":3},"fleets":{"default":[["at43",5],["B735",1],["B738",9]]}},{"name":"Emirates","icao":"uae","callsign":{"name":"Emirates","length":3},"fleets":{"90long":[["A306",7],["A310",14]],"90default":[["B733",1]],"default":[["A332",8],["A343",4],["A388",79],["A332",13],["B772",4],["B77L",10],["B773",12],["B77W",117]],"cargo":[["B77L",13],["B744",2]]}},{"icao":"ual","name":"United Airlines","callsign":{"name":"United","length":3},"fleets":{"90long":[["B762",15],["B752",15],["DC10",5],["B744",23]],"90default":[["B722",15],["B735",7],["B733",15]],"long":[["B744",24],["B752",96],["B753",21],["B763",35],["B764",16],["B772",22],["b77l",55],["B788",11]],"default":[[
```

Comment:
whoops.  this file should be removed.  should only be `airlines.json` in this folder